agent/taskresource/envFiles/envfile.go

```diff
@@ -332,6 +337,7 @@ func (envfile *EnvironmentFileResource) downloadEnvfileFromS3(envFilePath string
 	seelog.Debugf("Downlading envfile with bucket name %v and key name %v", bucket, key)
 	// we save envfiles to path: /var/lib/ecs/data/envfiles/cluster_name/task_id/${s3filename.env}
```

Comment:
So here the s3filename is the key name. Key is the unique identifier in the bucket. So what if for two different s3 files, they have the same key name, but from different buckets? I think we are just going to override each other here.

Reply:
good point, i will add bucket to path.